tests/acceptance/features/apiNotification/spaceNotification.feature

```diff
@@ -96,10 +96,7 @@ Feature: Notification
                     ]
                   },
                   "id": {
-                    "type": "string",
-                    "enim": [
-                      "%user_id%"
-                    ]
+                    "const": "%user_id%"
```

Comment:
failed here because we check Brians uuid instead of Alice uuid

error message if we use const: 
` Const failed at #->properties:messageRichParameters->properties:user->properties:id (Swaggest\JsonSchema\Exception\ConstException)`

error message if we use enum: 
`Enum failed, enum: ["4cd3bf36-af30-469d-ba2c-2090819acaf2"], data: "852b0ee2-75d9-4a98-84ef-1342b1566428" at #->properties:messageRichParameters->properties:user->properties:id (Swaggest\JsonSchema\Exception\EnumException)`